bindings/nodejs/Cargo.toml

```diff
@@ -18,7 +18,7 @@ exclude = [ "index.node" ]
 crate-type = [ "cdylib" ]
 
 [dependencies]
-iota-wallet = { path = "../../", default-features = false, features = [ "mnemonic", "events", "ledger_nano", "storage", "stronghold", "message_interface", "participation" ] }
+iota-wallet = { path = "../../", version = "1.0.0-rc.1", default-features = false, features = [ "mnemonic", "events", "ledger_nano", "storage", "stronghold", "message_interface", "participation" ] }
```

Comment:
Why ?

Reply:
Thanks for reviewing the PR, but it is still in draft mode, which is why it contains such things. Cargo added this when I was building the bindings, but I also think this is not needed, so I'll remove it.

Reply:
Sorry, my bad. Will wait for your green light 🟢